Example/Example/SeriesDetailViewController.swift

```diff
@@ -9,6 +9,92 @@
 import Foundation
 import BothamUI
 
-class SeriesDetailViewController : ExampleViewController {
+private struct Config {
+    static let numberOfColumns = 3
+    static let cellHeight = 174
+    static let headerHeight = 534
+    static let footerHeight = 20
+    static let cellMargin = 20
+}
+class SeriesDetailViewController : ExampleViewController, BothamCollectionViewController, SeriesDetailUI, UICollectionViewDelegateFlowLayout, UICollectionViewDelegate, UICollectionViewDataSource {
+
+
+    var dataSource: BothamCollectionViewDataSource<Comic, ComicCollectionViewCell>!
+    var header: Series?
+
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    override func viewDidLoad() {
+        configureNavigationBar()
+        configureCollectionView()
+        super.viewDidLoad()
+    }
+
+    override func viewWillDisappear(animated: Bool) {
+        navigationController?.navigationBar.setBackgroundImage(nil, forBarMetrics: UIBarMetrics.Default)
+        navigationController?.navigationBar.backgroundColor = UIColor.navigationBarColor
+        navigationController?.navigationBar.shadowImage = nil
+        navigationController?.navigationBar.translucent = false
+    }
+
+    func collectionView(
+        collectionView: UICollectionView,
+        numberOfItemsInSection section: Int) -> Int {
+            return self.dataSource.collectionView(collectionView, numberOfItemsInSection: section)
+    }
+
+    func collectionView(
+        collectionView: UICollectionView,
+        cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
+            return dataSource.collectionView(collectionView,
+                cellForItemAtIndexPath: indexPath)
+    }
+
+    func collectionView(collectionView: UICollectionView,
+        viewForSupplementaryElementOfKind kind: String,
+        atIndexPath indexPath: NSIndexPath) -> UICollectionReusableView {
+            if kind == UICollectionElementKindSectionHeader {
+                let headerView =
+                collectionView.dequeueReusableSupplementaryViewOfKind(kind,
+                    withReuseIdentifier: "SeriesDetailCollectionHeaderViewReusableIdentifier",
+                    forIndexPath: indexPath)
+                    as! SeriesDetailCollectionHeaderView
+                if let header = self.header {
+                    headerView.configureForItem(header)
+                }
+                return headerView
+
+            }
+            assert(false, "Unexpected element kind")
```

Comment:
I need to add that assert to get the code compiling. The other option is to return always the header even if the kind variable is associated to a UICollectionElementKindSectionFooter.